apps/src/lib/levelbuilder/lesson-editor/EditTipDialog.jsx

```diff
@@ -88,14 +98,21 @@ export default class EditTipDialog extends Component {
             defaultValue={this.state.tip.markdown}
             onChange={this.handleTextChange}
             style={styles.textArea}
+            rows={5}
           />
           <LessonTip tip={this.state.tip} />
         </div>
-        <DialogFooter rightAlign>
+        <DialogFooter>
+          <Button
+            __useDeprecatedTag
```

Comment:
What does this mean/do, and what non-deprecated thing should we ideally be doing instead? it would be good to understand this, to get an idea of whether these can be cleaned up as we go.

Reply:
Removed